addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/WebCacheDeceptionScanRuleUnitTest.java

```diff
@@ -28,9 +28,12 @@
 
 import fi.iki.elonen.NanoHTTPD;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Should this dependency be added specifically?

I think it's currently coming through from core.

https://github.com/pf-msi/zap-extensions/blob/issue-6655-webcachedeception-performance/addOns/ascanrulesAlpha/ascanrulesAlpha.gradle.kts

Reply:
That's fine, core exposes that one on purpose.